numpy/fft/helper.py

```diff
@@ -6,11 +6,9 @@
 
 import collections
 import threading
-
+import numpy as np
 from numpy.compat import integer_types
-from numpy.core import (
-        asarray, concatenate, arange, take, integer, empty
-        )
+from numpy.core import integer, empty, arange
```

Comment:
I think better to put `asarray` and `roll` here, rather than importing `numpy` - it cuts down on the import loops